### ISSUE TITLE
Selecting channel messages with conditions

### ISSUE THREAD

**Issue by yurir** [qeesung/rocketchat-node#72]
I need to select all messages after some date from some group.
I am trying to execute the following call:
```
rocketChatClient.groups.history({
    roomId: roomId
  }, {_updatedAt: {$gte: '2017-11-02'}}, function (err, body) {
    if (err)
      console.log(err);
    else {
      console.log('here comes body', body);
    }
  });
```
The snippet above selects all messages of the group, but ignores  date condition. 

Please advice.

**Comment by nickasd** [comment]
If your filter is `{_updatedAt: {$gte: '2017-11-02'}}`, you should replace that part with `{query: <filter>}`, as documented [here](https://github.com/qeesung/rocketchat-node#Channels.list).

**Comment by yurir** [comment]
hi @nickasd . i tried that, still doesn't work

**Comment by yurir** [comment]
resolved after upgrading rocket to 0.64.2
